lib/manager/python/extract.js

```diff
@@ -0,0 +1,16 @@
+
+module.exports = {
+  extractDependencies,
+};
+
+function extractDependencies(packageContent) {
+  const regex = /^([a-z][-\w.]+)==([\d.]+)/ig;
```

Comment:
Noting here that this initial support would be for exact deps only, e.g. not supporting pip range syntax

Reply:
I believe Python packages can start with an uppercase letter as per [PEP 508](https://www.python.org/dev/peps/pep-0508/#names). Can we just copy their regex? (Do they distribute a package with that regex?)

Reply:
I believe Python packages can start with ~an uppercase letter~ a number as per PEP 508. Can we just copy their regex? (Do they distribute a package with that regex?)

Reply:
We should probably only match those lines that contain nothing after the version until we have the appropriate logic to handle [pip hash checks](https://pip.pypa.io/en/stable/reference/pip_install/#hash-checking-mode). The hashes need to be updated in conjunction with the versions.